elevation_mapping/src/ElevationMapping.cpp

```diff
@@ -435,6 +445,30 @@ bool ElevationMapping::getSubmap(grid_map_msgs::GetGridMap::Request& request, gr
   return isSuccess;
 }
 
+bool ElevationMapping::getRawSubmap(grid_map_msgs::GetGridMap::Request& request, grid_map_msgs::GetGridMap::Response& response)
```

Comment:
This entire section is almost duplicate code from the `getSubmap` method. Also the naming is not clear. You should add an argument with default value to indicate fused or raw map access.

Reply:
Do you mean fix the service to return either fused map or raw map?
I think there should be two options available at the same time.
Some node may need fused submap and others may need raw submap.